docs/concepts.md

```diff
@@ -248,6 +248,20 @@ allowed (via two uni-directional Channels) if
 `(L_a ⊑ L_b) ∧ (L_b ⊑ L_a) ⇒ L_a = L_b`, i.e. if `a` and `b` have identical
 confidentiality and integrity.
 
+#### Node and Channel Creation
+
+Labels associated with Nodes and Channels are themselves effectively public, in
+the sense that they may be observed from outside the system.
+
+For this reason, when a Node attempts to create a new Node or Channel, an
+additional check that the label of the caller Node "flows to" the "public
+untrusted" label. This prevents the caller node from encoding non-public
+information in the label of the newly created Node or Channel, or even in the
+mere fact that such a Node or Channel exists at all.
+
+This restricts the ability to create new Node and Channels to Nodes that are
+themselves already "public".
```

Comment:
docs/abi.md could do with an update to mention this too.

Reply:
Done.